### PR TITLE
Completed prep-css-flexbox.

### PR DESCRIPTION
**Pull request by saraysc**
image
image
image
image

### CODE REVIEW

File: prep-css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck{
+      display: flex;
+      flex-basis: calc(100%/3);
```
Comment (thebearingedge) [inline prep-css-flexbox/align-items.html]: We should move this flex-basis to a rule set for `.card-wrapper`

File: prep-css-flexbox/flex-wrap.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck{
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```
Comment (thebearingedge) [inline prep-css-flexbox/flex-wrap.html]: We should move this flex-basis to the rule set for `.card-wrapper`

File: prep-css-flexbox/justify-content.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+<html>
+<head>
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck{
+      display: flex;
+      flex-basis: 25%;
```
Comment (thebearingedge) [inline prep-css-flexbox/justify-content.html]: We should move this flex-basis to the rule set for `.card-wrapper`

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Hi Sara, `flex-basis` only has an effect on "flex items" so we will need to make sure that property is only applied to the rule sets for `.card-wrapper`

File: prep-css-flexbox/justify-content.html (hunk 3 above)
Comment (saraysc) [reply prep-css-flexbox/justify-content.html]: Do I have to start all over again?

Comment (thebearingedge) [reply prep-css-flexbox/justify-content.html]: no, there's a guide for how to apply changes, the bot should have sent it to you

https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/pull-request-feedback

**Comment by saraysc** [comment]
image
image
image

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**